examples/tv-casting-app/linux/simple-app-helper.cpp

```diff
@@ -282,6 +316,9 @@ CHIP_ERROR PrintAllCommands()
     streamer_printf(sout, "  stop-discovery       Stop Discovery of Casting Players. Usage: cast stop-discovery\r\n");
     streamer_printf(
         sout, "  request <index>      Request connecting to discovered Casting Player with [index]. Usage: cast request 0\r\n");
+    streamer_printf(sout,
+                    "  request <index> cgp  Request connecting to discovered Casting Player with [index] using the "
+                    "Commissioner-Generated Passcode commissioning flow. Usage: cast request 0 cgp\r\n");
```

Comment:
In the pre-existing printf, can you clarify that it will trigger the "Commissionee-Generated Passcode commissioning flow"?